src/components/subscriptions/SubscriptionPlanRoutes.jsx

```diff
@@ -5,7 +5,7 @@ import { Route } from 'react-router-dom';
 import { connect } from 'react-redux';
 
 import MultipleSubscriptionsPage from './MultipleSubscriptionsPage';
-import SubscriptionDetailPage from './SubscriptionDetailPage';
+import SubscriptionDetailPageContainer from './SubscriptionDetailPage';
```

Comment:
does `Container` get appended to the component name when it is exported?

Reply:
This is the default export so you can name it anything. Appending container here since it's using the component connected to redux.

Reply:
An alternative name might be `ConnectedSubscriptionDetailPage`. OK with either, though.